### PR TITLE
Russolo: v1.000 added

### PR DESCRIPTION
**Pull request by laerm0**
Russolo: v1.000 added.

Taken from the upstream repo https://github.com/laerm0/Russolo at commit cec7d204da2d0e1c31fe80ebd2162f6b707e3148

### CODE REVIEW

File: ofl/russolo/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<p>
+Russolo is a typeface combining Bodoni and Futura into a high-contrast geometric sans.
+
+When I was a student at RIT in 2008-2010, I studied typography under the mentorship of Chuck Bigelow. I worked on a research project intended to find accurate revivals by comparing high-resolution scans of original prints to digitizations. Then it expanded to quantifying what the “average” typeface looked like from the main eras of typographic development by measuring various typefaces from those eras. Finally, I thought that it would be interesting to use my data to combine unique designs into something new (or at least interesting).
```
Comment (m4rc1e) [inline ofl/russolo/DESCRIPTION.en_us.html:4]: @laerm0 a double CR doesn't start a new paragraph in html. If you'd like each text block to be a new paragraph, do the following:

```
<p>
paragraph1
</p>
<p>
paragraph2
</p>
...
```

Comment (laerm0) [reply ofl/russolo/DESCRIPTION.en_us.html:4]: Basic HTML! A mystery. 😂

**Comment by laerm0** [comment]
Well, with `opsz` being a no-go, how should I revise this? `wght`-only VFs, but how many? `opsz` design space is 12-72.

**Comment by laerm0** [comment]
I'd be inclined to go 12, 20, 32, 48, 72.

**Comment by m4rc1e** [comment]
I'm not aware that "opsz" is a no go?

**Comment by laerm0** [comment]
#2866, #2870?

**Comment by laerm0** [comment]
You were on the thread...so I must be misunderstanding.

**Comment by m4rc1e** [comment]
> #2866, #2870?

We don't have these issue in this repo. Could you provide me with working links please.

**Comment by laerm0** [comment]
Whoops, I tried to autolink to a different repo :)

https://github.com/googlefonts/fontbakery/issues/2866

Isn't this a problem?

**Comment by m4rc1e** [comment]
This pr doesn't contain any fonts.

Add the fonts then run `gftools add-font` on the family dir please.

**Comment by laerm0** [comment]
Okay, ttf is in there now but `gftools add-font` is throwing an error at me. 

```
Traceback (most recent call last):
  File "/Users/stupakm/venvs/type-tools-py3/bin/gftools-add-font.py", line 267, in <module>
    app.run(main)
  File "/Users/stupakm/venvs/type-tools-py3/lib/python3.7/site-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/Users/stupakm/venvs/type-tools-py3/lib/python3.7/site-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "/Users/stupakm/venvs/type-tools-py3/bin/gftools-add-font.py", line 253, in main
    metadata = _MakeMetadata(fontdir, is_new)
  File "/Users/stupakm/venvs/type-tools-py3/bin/gftools-add-font.py", line 122, in _MakeMetadata
    font_license = fonts.LicenseFromPath(fontdir)
  File "/Users/stupakm/venvs/type-tools-py3/lib/python3.7/site-packages/gftools/util/google_fonts.py", line 684, in LicenseFromPath
    return _EntryForEndOfPath(path, _KNOWN_LICENSE_DIRS)
  File "/Users/stupakm/venvs/type-tools-py3/lib/python3.7/site-packages/gftools/util/google_fonts.py", line 668, in _EntryForEndOfPath
    raise ValueError('Found %d possible matches: %s' % (len(answers), answers))
ValueError: Found 0 possible matches: []
```

It's also looking for `.nam` files though, for some reason. I run `gftools add-font russolo` from the `fonts/ofl` directory and I get a bunch of warnings like `no cp file for yi found at /yi_unique-glyphs.nam` before the error above.

**Comment by m4rc1e** [comment]
It wants a license. Include an OFL.txt.

**Comment by laerm0** [comment]
The OFL is in there and committed. That's not it.

**Comment by RosaWagner** [comment]
Hello, what is the status of this PR ? Is there something that needs to be done here or upstream?

**Comment by m4rc1e** [comment]
Will close this PR since the family needs further work

**Pull request closed without merging**